EventFilter/HcalRawToDigi/src/HcalUnpacker.cc

```diff
@@ -856,5 +856,20 @@ void HcalUnpacker::unpack(const FEDRawData& raw, const HcalElectronicsMap& emap,
       }
     }
   }
-}      
+}
+// Method to unpack uMNio data
+void HcalUnpacker::unpackUMNio(const FEDRawData& raw, int slot, HcalUMNioDigi& umnio) {
+  const hcal::AMC13Header* amc13=(const hcal::AMC13Header*)(raw.data());
+  int namc=amc13->NAMC();
+  //Find AMC corresponding to uMNio slot
+  for (int iamc=0; iamc<namc; iamc++) {
+    if (amc13->AMCSlot(iamc) == slot) namc = iamc;
+  }
+  if (namc==amc13->NAMC()) return;
+  
+  const uint16_t* data = (uint16_t*)(amc13->AMCPayload(namc));
```

Comment:
static code analysis raises an issue here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14461/12931/llvm-analysis/report-f0dfaa.html#EndPath
this const cast-away is not needed, apparently. Can we change to 
```C++
const uint16_t* data = (const uint16_t*)(amc13->AMCPayload(namc));
```
?